eml_parser/eml_parser.py

```diff
@@ -120,6 +129,10 @@ def __init__(self,
         self.ignore_bad_start = ignore_bad_start
         self.email_force_tld = email_force_tld
         self.parse_attachments = parse_attachments
+        self.include_www = include_www
+        self.include_href = include_href
+        self.valid_domain_or_ip = valid_domain_or_ip
+        self._psl = publicsuffixlist.PublicSuffixList(accept_unknown=not self.email_force_tld)
```

Comment:
Though we are starting to add a lot of flags, I think it might be better to use a new flag for the PSL feature instead of re-using "email_force_tld" for that ?